jupyterlab_comments/package.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
We've tried to keep the package.json file consistent between extensions so that the `lerna` commands at the top of the repository work properly for all subpackages.

https://github.com/GoogleCloudPlatform/jupyter-extensions/blob/master/jupyterlab_gcpscheduler/package.json is a good example. The scripts there serve mostly the same purpose but many can be called from the top of the repo to run a build or a test in parallel for each subpackage.

You can then add this subfolder to https://github.com/GoogleCloudPlatform/jupyter-extensions/blob/master/lerna.json to have it be included.